src/components/WeatherComponents/Weather.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import styles from './Weather.module.css';
+import axios from 'axios';
+import { apiUrl } from './Weather.settings';
+
+interface WeatherData {
```

Comment:
interfaces that are not part of the props should be in interfaces folder in seperate files